apps/serialization/messages.proto

```diff
@@ -171,17 +175,17 @@ enum GameStatus {
  * is_moving: true if the entity is moving
  */
 message Entity {
-  uint64 id = 1;
-  string category = 2;
-  string shape = 3;
-  string name = 4;
+  optional uint64 id = 1;
+  optional string category = 2;
+  optional string shape = 3;
+  optional string name = 4;
   Position position = 5;
-  float radius = 6;
-  repeated Position vertices = 7;
+  optional float radius = 6;
+  optional ListPositionPB vertices = 7;
```

Comment:
This is an interesting change that is needed for diffing lists under a specific scenario, otherwise its a near impossible task. So the scenarios
- If `new_list` is different than `old_list` (meaning not an exact match) the resulting delta is `new_list` 
- If `new_list` is exactly the same as `old_list` the resulting delta is `nil` (no delta)

To allow for that second scenario we would have needed a sort of `optional repeated` field in protobuf, but this is not supported. So the second best thing is this solution where we wrap the list in another message. This wrapper becomes `optional` and the inner only have the `repeated` field